test/Microsoft.ML.Tests/Scenarios/Api/AspirationalExamples.cs

```diff
@@ -61,4 +61,89 @@ public void FirstExperienceWithML()
+            CheckSchemaHasColumn(schema, "area", out int areaIdx);
+            CheckSchemaHasColumn(schema, "title", out int titleIdx);
+            CheckSchemaHasColumn(schema, "description", out int descriptionIdx);
+
+            // Next verify they have the expected types.
+            Assert.Equal(TextType.Instance, schema.GetColumnType(areaIdx));
+            Assert.Equal(TextType.Instance, schema.GetColumnType(titleIdx));
+            Assert.Equal(TextType.Instance, schema.GetColumnType(descriptionIdx));
+
+            // sefilipi: Do we need to inspect the data, as in Tom's example?	
+
+            var estimator = Estimator.MakeNew(data)
+                .Append(row => (
+                    // Convert string label to key. 
+                    label: row.area.Dictionarize(),
+                    // featurizes 'description'
+                    description1: row.description.FeaturizeText(),
+                    // featurizes 'title'
+                    title1: row.title.FeaturizeText()))
+                .Append(row => (
+                    // Concatenate the two features into a vector.
+                    // Do we need to make other use of the TextTransform output columns?
+                    features: row.description1.ConcatWith(r.title1),
+                    // preserve the label
+                    label: row.label))
+                // how do we specify arguments for the trainer?
+                .Append(row => r.label.TrainLinearClassification(row.Features));
+
+            var model = estimator.Fit(data);
+
+            string modelPath = "github-Model.zip";
+
+            // we don't currently have the WriteAsync
+            await model.WriteAsync(modelPath);
+        }
+
+        public void PredictGithubIssues()
+        {
+            ClassifyGithubIssues();
+
+            // we don't currently have the ReadAsync
+            var model = await PredictionModel.ReadAsync(ModelPath);
+
+            //create an example issue:
+            const string data = "29338	" +
+                                "area-System.Net 	" +
+                                "Include fragment and query in Uri.LocalPath on Unix	" +
+                                "While testing XmlUriResolver, @pjanotti discovered that any segments of a file path following a '#' symbol will be cut out of Uri.LocalPath on Unix. Based on additional tests, this also occurs for the '?' symbol. This is happening because the Unix specific case for local path only uses the path component of the URI:  https://github.com/dotnet/corefx/blob/9e8d443ff78c4f0a9a6bedf7f95961cf96ceff0a/src/System.Private.Uri/src/System/Uri.cs#L1032-L1037    The fix here is to include the fragment and query in LocalPath in the Unix path specific case. This PR enables the test case in XmlUriResolver that uncovered this issues, and adds some additional cases to our URI tests.    Fixes: #28486 ";
+            var dataSource = new BytesStreamSource(data);
```

Comment:
>BytesStreamSource [](start = 33, length = 17)

I think this sends the wrong message. In one-off prediction, you don't need `data source`, you just have one example.

We should have something like 
```c#
var predictor = model.MakePredictionFunction<IssueInput, IssuePrediction>();
var result = predictor.Predict(new IssueInput(...)
```